File: JavaScript/1.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */

let twoSum = (nums, target) => {
  for (let i = 0; i < nums.length - 1; i++) {
    for (let j = i + 1; j < nums.length; j++) {
      if (nums[i] + nums[j] == target) return [i, j];
    }
  }
};

File: Python/1.py
class Solution:
    def twoSum(self, nums, target):  
        list = []
        for i in range (0, len(nums) - 1):
            for j in range (i + 1, len(nums)):
                if nums[i] + nums[j] == target:
                    list.append(i)
                    list.append(j)
        return list


nums = [2, 7, 11, 15]
target = 9

print(Solution().twoSum(nums, target))
